### DISCUSSION TITLE
Can I not read the template file every time?

### DISCUSSION THREAD

**Discussion by jincandev** [frandiox/vite-ssr#135] [Ideas]
https://github.com/frandiox/vite-ssr/blob/eabb2dfcac5574c8db35bfa809f903ea4fd275b4/src/dev/server.ts#L56

Can I not read the template file every time?

**Comment by frandiox** [comment]
It is read every time because the `index.html` file could have been modified during development. This guarantees getting the latest version.
It can be optimized by adding a watcher and only reading it if the file has actually changed. However, I think I prefer the current simpler version (this is only happening in development anyway, not in prod).

Is this giving you any problem? Reading that file shouldn't be too slow 🤔

**Reply by jincandev** [reply]
No problem. I just compared the way I am using with the way you write here, so this problem is extended. I now realize that this is just a different strategy.
